doc/index.rst

```diff
@@ -211,21 +219,30 @@ You can pull any `Docker` image by specifying in `DOCKER_IMAGE` variable, as lon
 
 If your Docker image is missing any of the above libraries, then you can still pass their name by `ADDITIONAL_DEBS` (see `variables section <./index.rst#optional-environment-variables>`_).
 
-Note-1. This disables the handling of `ROS_REPOSITORY_PATH` and `ROS_DISTRO` as ROS needs already to be installed in the image.
+Some more notes:
+- Specifying Docker image disables the handling of `ROS_REPO` (and non-recommended `ROS_REPOSITORY_PATH`), and `ROS_DISTRO` as ROS needs to be installed in the image.
+- For some images, `ROS_DISTRO` variable still needs to be set. This holds for `ROS official Docker images <https://hub.docker.com/_/ros/>`_ as of Sept. 2017.
```

Comment:
I'm not sure if these 2 lines hold true.

Reply:
This is a little bit tricky:

> Specifying `DOCKER_IMAGE`  disables the set-up of ROS based on `ROS_REPO` (or non-recommended `ROS_REPOSITORY_PATH`), and `ROS_DISTRO`, but `ROS_DISTRO` still needs to be set.

Reply:
Regarding the the section above this note:

> You can pull any `Docker` image by specifying in `DOCKER_IMAGE` variable, as long as the following requirement is met:
>  
>  * sources.list set up (`example <http://wiki.ros.org/kinetic/Installation/Ubuntu#Installation.2BAC8-Ubuntu.2BAC8-> Sources.Setup_your_sources.list>`_).
> * `python-catkin-tools`, `python-pip`, `python-rosdep`, `python-wstool`, `ros-$ROS_DISTRO_catkin`
> If the image does not contain these packages, they can get installed by including them in `ADDITIONAL_DEBS`, too.